srcpkgs/volumeicon/template

```diff
@@ -1,14 +1,20 @@
 # Template file for 'volumeicon'
 pkgname=volumeicon
 version=0.5.1
-revision=2
+revision=3
 build_style=gnu-configure
 configure_args="--enable-notify"
-hostmakedepends="pkg-config intltool"
-makedepends="alsa-lib-devel libnotify-devel gtk+3-devel"
+# automake, glib-devel and gettext-devel are needed for pre_configure
+hostmakedepends="pkg-config intltool automake glib-devel gettext-devel"
+makedepends="alsa-lib-devel libnotify-devel gtk+3-devel glib-devel gettext-devel"
 short_desc="Lightweight volume control that sits in your systray"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3"
 homepage="http://softwarebakery.com/maato/volumeicon.html"
 distfiles="http://softwarebakery.com/maato/files/volumeicon/volumeicon-${version}.tar.gz"
 checksum=24b8c1d0a81d708b201ce6e67301fc175d65588d892d01859f667b8db8a05da0
+
+# necessary because of notify patch
+pre_configure() {
+    autoreconf -fi
```

Comment:
Please use TAB for indentation.

Reply:
Anyway, there's something I would like to change:
```diff
diff --git a/srcpkgs/volumeicon/template b/srcpkgs/volumeicon/template
index 130cb164cd..ae6483e590 100644
--- a/srcpkgs/volumeicon/template
+++ b/srcpkgs/volumeicon/template
@@ -4,17 +4,26 @@ version=0.5.1
 revision=3
 build_style=gnu-configure
 configure_args="--enable-notify"
-# automake, glib-devel and gettext-devel are needed for pre_configure
 hostmakedepends="pkg-config intltool automake glib-devel gettext-devel"
-makedepends="alsa-lib-devel libnotify-devel gtk+3-devel glib-devel gettext-devel"
+makedepends="alsa-lib-devel libnotify-devel gtk+3-devel libglib-devel"
 short_desc="Lightweight volume control that sits in your systray"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-3"
+license="GPL-3.0-only, ISC, MIT"
 homepage="http://softwarebakery.com/maato/volumeicon.html"
 distfiles="http://softwarebakery.com/maato/files/volumeicon/volumeicon-${version}.tar.gz"
 checksum=24b8c1d0a81d708b201ce6e67301fc175d65588d892d01859f667b8db8a05da0
 
-# necessary because of notify patch
+post_patch() {
+	sed -ne '/Copyright/,/IN THE SOFTWARE/s/ [*] *//p' src/bind.c >LICENSE.MIT
+	sed -ne '/Copyright/,/OF THIS SOFTWARE/s/ [*] *//p' \
+		src/alsa_volume_mapping.c >LICENSE.ISC
+}
+
 pre_configure() {
-    autoreconf -fi
+	autoreconf -fi
+}
+
+post_install() {
+	vlicense LICENSE.MIT
+	vlicense LICENSE.ISC
 }
```